app/models/container_template.rb

```diff
@@ -1,5 +1,6 @@
 class ContainerTemplate < ApplicationRecord
   include CustomAttributeMixin
+  include ManageIQ::Providers::Kubernetes::ContainerManager::EntitiesMapping
```

Comment:
@zakiva @moolitayer the dependency direction introduced in this PR is not the correct one. It should be the other way around.